umbral-pre-wasm/pkg-template/.gitignore

```diff
@@ -0,0 +1,2 @@
+pkg-bundler
```

Comment:
I don't think this is necessary here, `pkg-template` never has these subfolders, they're created in `pkg`.